src/lib/metrics.js

```diff
@@ -1,32 +1,51 @@
+/* global Metrics */
```

Comment:
This `Metrics` comes from testpilot-metrics, included as a script alongside `background.js` in the `manifest.json`